web/docs/getting-started.md

```diff
@@ -53,6 +53,17 @@ We recommend using [nvm](https://github.com/nvm-sh/nvm) for managing your Node.j
   </div>
 </details>
 
+:::info
+
+Why does Wasp require this specific `node` range and doesn't support a newer version x.y.z?
+
+At Wasp, we focus on supporting the latest LTS ("long-term-support") Node.js version, since it guarantees stability and active maintainance, which is why official Node.js team recommends it for usage in production.
```

Comment:
"for usage in production" -> "using it in production" - maybe that is nicer/simpler?